### ISSUE TITLE
Mini/NPC Character Sheet: Description pane too small.

### ISSUE THREAD

**Issue by Kirzai** [crnormand/gurps#135]
Discovered while attempting to populate the sheet for a ghoul with this description:
_A ghoul appeared as an emaciated, roughly humanoid creature with an almost-hairless mottled, decaying hide stretched tight over its bones. It had the sharp teeth of a carnivore and sunken eyes that burned as if they were hot coals._
I would consider this a fairly short description for a monster, and I'd like to eventually populate my game world with a large self-contained monster compendium, and that would require significantly more space for descriptive text.

**Comment by Kirzai** [comment]
Discovered while attempting to populate the sheet for a ghoul with this description:
_A ghoul appeared as an emaciated, roughly humanoid creature with an almost-hairless mottled, decaying hide stretched tight over its bones. It had the sharp teeth of a carnivore and sunken eyes that burned as if they were hot coals._

**Comment by mjeffw** [comment]
Added to the backlog for consideration. For now, the workaround would be to use the Notes area.

**Comment by mjeffw** [comment]
Implemented, and will go out in a future update. I changed the editor and full sheet to display the description in a <textarea>, which allows you to see multiple lines and does the expected word wrapping, etc. I tweaked the layout in the NPC sheet to grow to be large enough to show the whole description. Images attached.

Editor:
![Screen Shot 2020-12-28 at 5 21 04 PM](https://user-images.githubusercontent.com/7349413/103246321-133e3080-4931-11eb-9b3d-c0803ed268c1.png)

Full Sheet:
![Screen Shot 2020-12-28 at 5 21 39 PM](https://user-images.githubusercontent.com/7349413/103246358-31a42c00-4931-11eb-8172-9d87e39a006f.png)

NPC Sheet:
![Screen Shot 2020-12-28 at 5 22 42 PM](https://user-images.githubusercontent.com/7349413/103246403-4d0f3700-4931-11eb-8af1-db5f9631164c.png)
